skins/moono-lexicon/presets.css

```diff
@@ -9,6 +9,15 @@ For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
 	display: inline;
 }
 
+/* Hide button labels on 768px breakpoint */
+@media (max-width: 768px) {
```

Comment:
Is this what we want to do?

> the goal of this story is to limit the information that is shown in CKEditor's toolbar when the space allowed is not enough by hiding the buttons text.

With the media-query we're only looking for the window size, not the size of the editor. If the editor's container is smaller than 768 but the window is not this is result:

![image](https://user-images.githubusercontent.com/5803434/87047969-74474000-c1fb-11ea-8e00-0648eec8d8cc.png)

Reply:
![this](https://user-images.githubusercontent.com/7074/87134841-84adf800-c299-11ea-9a5f-bbb1bd96f8f5.gif)

Reply:
> Is this what we want to do?
> 
> > the goal of this story is to limit the information that is shown in CKEditor's toolbar when the space allowed is not enough by hiding the buttons text.
> 
> With the media-query we're only looking for the window size, not the size of the editor. If the editor's container is smaller than 768 but the window is not this is the result:
> 
> ![image](https://user-images.githubusercontent.com/5803434/87047969-74474000-c1fb-11ea-8e00-0648eec8d8cc.png)

The other way, of getting the editor container size instead of using media queries would me tricky. 
I think this solution would be sufficient in this case. 
This way we handled the responsive behaviour and resizing of editor respectfully in relation with window size and visibility of button labels which falls in the acceptance criteria.

Reply:
Hey @mateomustapic, while this might be a good compromise, I agree with @carloslancha that this is not enough.

One possible solution to avoid media-queries and still keep a simple implementation would be:
- On resize, add custom breakpoint classes based on editor size (`editor-sm`, `editor-lg`, `editor-xl`)
- Use those classes instead of media queries to apply specific styles to breakpoint sizes

That still doesn't account exactly for "not having enough space" but I think would be more acceptable than relying on window size in general.

Reply:
> The other way, of getting the editor container size instead of using media queries would me tricky and unnecessary.
> I think this solution would be sufficient in this case.
> This way we handled the responsive behaviour and resizing of editor respectfully in relation with window size and visibility of button labels which falls in the acceptance criteria.

I think you're misreading the acceptance criteria:

> As described in [Hiding Text](https://docs.google.com/document/d/1UPxckZRglY39i0UK6jEViv16C34HGRkTWxoCwqcr-kY), the goal of this story is to limit the information that is shown in CKEditor's toolbar when the space allowed is not enough by hiding the buttons text.
>
> Acceptance Criteria
> When the editor is resized and it starts wrapping
> Then Buttons no longer show labels inside them and only their icons remain

Let's not ship a half-baked fix that doesn't address the actual problem.

Reply:
> Hey @mateomustapic, while this might be a good compromise, I agree with @carloslancha that this is not enough.
> 
> One possible solution to avoid media-queries and still keep a simple implementation would be:
> 
> * On resize, add custom breakpoint classes based on editor size (`editor-sm`, `editor-lg`, `editor-xl`)
> * Use those classes instead of media queries to apply specific styles to breakpoint sizes
> 
> That still doesn't account exactly for "not having enough space" but I think would be more acceptable than relying on window size in general.

Ok, I will work on this suggested solution, thanks!